src/URI.js

```diff
@@ -1223,6 +1227,26 @@
   };
 
   // compound accessors
+  p.origin = function(v, build) {
+    var parts;
+
+    if (this._parts.urn) {
+      return v === undefined ? '' : this;
```

Comment:
When looking at [URN Examples](https://en.wikipedia.org/wiki/Uniform_Resource_Name#Examples) I wonder if the origin of `urn:isan:0000-0000-9E59-0000-O-0000-0000-2` would be `urn:isan`. But since anything but the protocol is considered to be the path, this wouldn't be trivial…